drafts/2016-01-06.md

```diff
@@ -0,0 +1,91 @@
+
+Not much else to report this week; a lot of people are off to enjoy CCC, and the holidays.
+
+## December 7th
+
+Here are some highlights of what happened during the [December 7 Sprint](https://github.com/ipfs/pm/issues/67) :
+
+### Releases
+* [@whyrusleeping](//github.com/whyrusleeping)  shipped IPFS version 0.3.10!  It contains [74 new commits](https://github.com/ipfs/go-ipfs/compare/v0.3.9...v0.3.10) since the previous version and you can [get it here](https://ipfs.io/docs/install/).
+* npm on IPFS!  `registry-mirror` is a new tool that enables distributed discovery of npm modules by fetching and caching the latest state of npm through IPNS.  For more info, see this [blog post](http://blog.daviddias.me/2015/12/08/stellar-module-management) by [@diasdavid](//github.com/diasdavid) .
+* [@jbenet](//github.com/jbenet) released a new tool/library called [dnslink](https://github.com/jbenet/go-dnslink) that makes it easy to resolve dns links (special TXT records in a domain name that can point to paths, like an IPFS path)
+
+### Updates
+* [**(infrastructure)**](https://github.com/ipfs/infrastructure) On the infrastructure side of things, [@lgierth](//github.com/lgierth) has [bootstrapped](https://github.com/ipfs/infrastructure/pull/135) two new storage, each with 17 TB of disk space!  
+* [**(api)**](https://github.com/ipfs/api) [@RichardLitt](//github.com/RichardLitt) has [reached a draft 1]((https://github.com/ipfs/api/pull/13) of the much needed API documentation.
+* [@harlantwood](//github.com/harlantwood) wrote a bit of [nodejs code](https://github.com/ipfs/project-repos/pull/11) that spins up a fresh IPFS node, sets it to a known ID, and publishes to IPNS using that node 
+* [**(specs)**](https://github.com/ipfs/specs) The new IPFS Linked Data (IPLD) spec is actively being iterated on in the specs repository.  Join the discussion [here!](https://github.com/ipfs/specs/pull/37)
+
+### Active stuff
+* [@robcat](//github.com/robcat) and [@fazo](//github.com/fazo)96 have done great work integrating IPFS with pacman (the package manager for Arch Linux).  They can now install arch packages straight from IPFS!  For more details, see [this active discussion](https://github.com/ipfs/notes/issues/84).
+* [@Dignifiedquire](//github.com/Dignifiedquire) has been working on an attractive new distribution page for IPFS, which will be the new landing page to download all things IPFS.  You can see the [latest screenshots here](https://github.com/ipfs/distributions/issues/11).
+
+## Contributors
+
+Across the entire IPFS github organization, the following people have comitted code since December 7th. (We're autogenerating this list using [this tool](https://github.com/ipfs/weekly/pull/5), so please let us know if your name isn't here.) In the future, we will also include people who comment, as they are also super important; bear with us while we develop that technology.
+
+
+* Adam Gashlin
```

Comment:
agreed.  the username comes from git, so i consider it to be what the user would like to be publicly identified as.  the only mapping done is when it is obvious that a nickname is an alias for a public real name.

we don't list the github handles at the moment because we don't have a good way to get them.  this will be fixed in the future though